content/integrations/sqlserver.md

```diff
@@ -6,14 +6,62 @@ kind: integration
 doclevel: basic
 ---
 
+{{< img src="integrations/sql_server/sql_server_graph.png" alt="sql server graph" >}}
+
 ## Overview
 
 Connect SQL Server to Datadog in order to:
 
   * Visualize your database performance.
   * Correlate the performance of SQL Server with the rest of your applications.
 
-{{< insert-example-links conf="sqlserver" check="sqlserver" >}}
+<div class="alert alert-warning">
+SQL Server check can only be run from a Windows environment
+</div>
+
+## Configuration
+This integration requires a Datadog Agent version >= 3.2.3
+
+1. Make sure that your SQL Server instance supports SQL Server authentication by enabling "SQL Server and Windows Authentication mode" in the server properties. 
```

Comment:
If you look in integrations-core (e.g. for the MySQL check), I've been doing two sub-headers for configuration instructions like this (i.e. integrations where you first need to prepare the collected-from thing before connecting the Agent up):

### Prepare SQL Server

Make Sure your SQL Server instance supports...

Then create a read-only user for the Datadog Agent to use to collect metrics...

### Connect the Agent

Create a file `sqlserver.yaml` in the Agent's conf.d directory:

```
...
```

Restart the Agent to start sending SQL Server metrics to Datadog.